python/ray/tune/integration/wandb.py

```diff
@@ -40,7 +40,24 @@ def _clean_log(obj):
         return obj
```

Comment:
would it be better just to return `obj` if `obj` is instance of `int, float, np numerical, str` before this try-catch?

Reply:
That makes sense. Updated